notebooks/Add GPS Metadata Fields.ipynb

```diff
@@ -11,26 +11,37 @@
     "- Use this script to update an existing line or polygon feature class to support GPS metadata\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.                    if not set([0, 1, 2, 4, 5]).issubset(values):
</pre><p>Suggestion: You can directly use a set here:</p><p><code>if not {0, 1, 2, 4, 5}.issubset(values):</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Updated